src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -287,20 +263,25 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         return this._stripeElement;
     }
 
-    private _getStripeCardElements(): StripeCardElements {
-        if (!this._stripeCardElements) {
+    private _getStripeJs(): StripeV3Client {
+        if (!this._stripeV3Client) {
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        return this._stripeCardElements;
+        return this._stripeV3Client;
     }
 
-    private _getStripeJs(): StripeV3Client {
-        if (!this._stripeV3Client) {
-            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+    private _handleEmptyPaymentIntentError(
+        error: Error,
+        stripeError: StripeError | undefined
```

Comment:
What instance does it return `undefined`?

Reply:
`handleCardAction` return either a payment intent or an error (mutually exclusive)